docs/messaging-service/email-config.md

```diff
@@ -0,0 +1,32 @@
+# Email gateway configuration
+
+The Messaging service supports 3 ways of sending email messages out of the box:
+
+- By integrating with the OneWelcome email gateway
+- By integrating with an SMTP server
+- By integrating with a custom API
+
+## OneWelcome email gateway integration configuration
```

Comment:
is the word "integration" necessary here? it's not present in the other 2 points

Reply:
Not really, I'll remove it to be consistent with other points